Softeq.XToolkit.WhiteLabel.iOS/AppDelegateBase.cs

```diff
@@ -17,15 +18,15 @@ public abstract class AppDelegateBase : UIApplicationDelegate
     {
         public override UIWindow Window { get; set; } = default!;
 
-        protected abstract IBootstrapper Bootstrapper { get; }
-
         public override bool FinishedLaunching(UIApplication application, NSDictionary launchOptions)
         {
             InitializeWhiteLabelRuntime();
 
             return true;
         }
 
+        protected abstract IBootstrapper CreateBootstrapper();
```

Comment:
May be better to use as a property
Point to discuss with the team.

Reply:
As I understand, we don't really need to keep an instance of the Bootstrapper once it is initialized. And if it is kept - it is possible to use it second time (I believe that's why there was a check for "isInitialized" [here](https://github.com/Softeq/XToolkit.WhiteLabel/pull/317/files#diff-1584b30f91f5c1bf8f3953bc498d73b3L25))

Reply:
I agree.